### PR TITLE
Creates Collections docs

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

**Comment by bstuder99** [comment]
Have not added images yet, still waiting for some improvements in the demo to show a dashboard of multiple collections with K8s/cloud costs in them, which I currently can't do

File: using-kubecost/navigating-the-kubecost-ui/collections.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+![Collections](/images/collections.png)
+
+Kubernetes costs, also referred to as in-cluster costs, refer to spending on resources including nodes, disks, and network costs. They are monitored via the Assets dashboard.
+Cloud costs, also referred to as out-of-cluster or external costs, refer to spending on third party services such as services offered by cloud service providers. They are monitored via the Cloud Cost Explorer.
+
+## Creating a new collection
+
+To begin, select _Add a new Collection_ in the top right. If you have not created any collections yet, you should see a _Add a Collection_ icon in the middle of your screen. You can select this instead. The ‘New Collection’ page opens.
+The headline for your page should be an auto-generated title. This will be the name of your collection, which you can change by selecting the pencil icon. You can provide a name and also a kind for your collection. Kind is a custom label you can add to multiple collections to allow for filtering of that label, if multiple collections are intended for the same team or function. Name and kind can always be edited later.
```
Comment (aligon) [inline using-kubecost/navigating-the-kubecost-ui/collections.md:13]: We are re-labeling `kind` to `category` in the UI

File: using-kubecost/navigating-the-kubecost-ui/collections.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+![New Collection](/images/newcollection.png)
+
+All Kubernetes and cloud spend sources will appear on this page under one of two columns: Costs in Collection, or Costs Not in Collection. When first creating your collection, there should be no costs already added. Therefore, you should select _Costs Not in Collection_ to view a list of all available cost sources, or select _Explore Costs_.
+
+Costs are then divided into two categories: _Kubernetes_ and _Cloud_ (see above for an explanation of these costs). You can toggle between the two groups using the Domain dropdown on the left of the page. For each group, you will see a total cost value displayed on the right of the page, above your cost table.
+
+Costs are organized in a table, listed in descending order starting with the highest values. To add an individual item, hover your cursor over the item until you see a green _Add_ button visible on the right of the page. Select _Add_ to include that cost into your collection. You can add all your Kubernetes or cloud costs into a collection by selecting _Add All_, displayed next to your total cost. Once an item has been added to the collection, changes will be automatically saved.
```
Comment (aligon) [inline using-kubecost/navigating-the-kubecost-ui/collections.md:23]: There are certain scenarios they wouldn't be able to use `Add All`.

Up to you if thats worth calling out here.

Comment (bstuder99) [reply using-kubecost/navigating-the-kubecost-ui/collections.md:23]: Can you elaborate on why this would be?

Comment (aligon) [reply using-kubecost/navigating-the-kubecost-ui/collections.md:23]: I guess really only if there are no filters set, the others I saw were bugs that should be fixed.

**Comment by bstuder99** [comment]
@aligon I've made updates including to the Advanced Reporting pages, this should be ready to go

**Review by aligon — APPROVED** [review]

**Pull request merged**